src/Shared/UnitTests/Vacancies.Client/Application/CommandHandlers/PatchVacancyTrainingProviderCommandHandlerTests.cs

```diff
@@ -0,0 +1,116 @@
+using FluentAssertions;
+using Xunit;
+using Moq;
+using Microsoft.Extensions.Logging;
+using Esfa.Recruit.Vacancies.Client.Application.CommandHandlers;
+using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Application.Commands;
+using System.Threading.Tasks;
+using System.Threading;
+using System;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Services.TrainingProvider;
+
+namespace Esfa.Recruit.UnitTests.Vacancies.Client.Application.CommandHandlers
+{
+    [Trait("Category", "Unit")]
+    public class PatchVacancyTrainingProviderCommandHandlerTests
+    {
+        private const long EsfaUkprn = 10033670;
+        private const long MockedProviderUkprn = 12345678;
```

Comment:
Instead of MockedProviderUkprn can we rename this to NonEsfaUkprn. This makes the intent more clearer.

Reply:
👍